tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -492,18 +493,32 @@ generic_config_updater/test_incremental_qos.py::test_incremental_qos_config_upda
   skip:
     reason: "This test was not run on this hwsku type currently"
     conditions:
+      - "branch in ['internal-202205']"
       - "not any(i in hwsku for i in ['2700', 'Arista-7170-64C', 'montara', 'newport'])"
 
+generic_config_updater/test_mmu_dynamic_threshold_config_update.py::test_dynamic_th_config_updates:
+  skip:
+    reason: "This test behavior to be modified in 202205"
```

Comment:
Does this `skip` infra support to track the expected PR merged? so it will auto not skip if the condition met.

Reply:
Discussed offline. Since this skip condition will be removed when a second sonic-mgmt PR is merged to update these test cases, I added a TODO marker in the comments